.github/workflows/release.yml

```
@@ -76,8 +75,12 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           path: bin
+      - name: chmod executable
```

Just something more natural English and explanatory.

```suggestion
      - name: Make retrieved artifact executable
```

---

@mcdonnnj resolved with [d90443c](https://github.com/cisagov/con-pca-tasks/pull/7/commits/d90443c9a8faafafa6ce2c5ffeb009a7f3ef34c5)